compare.py

```diff
@@ -473,7 +506,9 @@ def sub_process_path(source_stable, source_dev, value):
             sub_result_list.append(base_count)
             sub_result_list.append(dev_count)
             try:
-                sub_result_list.append(f'{((dev_count - base_count) / base_count) * 100}%')
+                percentage_change = ((dev_count - base_count) / base_count) * 100
+                sub_result_list.append(f'{percentage_change}%')
+                total_per = total_per + percentage_change
             except:
```

Comment:
Dry except is not preferred. Use except Exception if we're not using the exception.

Reply:
@abhstabs Here, expect serves the goal of slowing the change to 0% when we are unable to determine the percentage change.

Reply:
use `except Exception` instead of a regular `except` statement. 

Dry excepts without exception(using except instead of except Exception) are not preferred in Python.

Reply:
okay okay got it, will made the change. thanks

Reply:
@abhstabs Done.